lib/action.ts

```diff
@@ -138,7 +138,25 @@ const ACTIONS: Record<ActionType, Actions> = {
 			"https://img2.gelbooru.com/images/94/83/94833376ed7271f08c75385204fbcd1f.gif",
 			"https://img.rule34.xxx//images/1149/9362d2e884efc6e49b665be3268d4f02d66674a3.gif?1148722",
 		]
+	},
+	kill: {
+		title: "You have died.",
+		message: "You feel the life slowly draining from your body as your draw in your last breaths.",
```

Comment:
The way `message` acts is like a verb, and the two people involved are placed either side of this message.

e.g  Person1 `messaage` Person2!  (`transitiveness: true`)
or Person2 `message` from Person1 (`transitiveness: false`)

Yeah?